### DISCUSSION TITLE
Date range field?

### DISCUSSION THREAD

**Discussion by Coysh** [solspace/craft-freeform#1382] [Questions & Answers]
I've got a form where I need a start date and end date. At the moment rhere are two date fields, but I can't prevent the user from selecting an end date before the start date.

Any ways to do this?

**Comment by kjmartens** [comment]
While the Date & Time field type allows setting min/max dates to be selected, there isn’t any built-in support for the ability to have two Date & Time fields to be dependent on each other to control that at this time.

However, it is probably something you could build a custom field type to handle if need be. You could use the Freeform Date field as a starting point (every Freeform field type is built the same way a 3rd party one would be). 🙂 

https://docs.solspace.com/craft/freeform/v5/developer/field-types/
